### DISCUSSION TITLE
Sensore non a contatto di rilevamento persona su sedia

### DISCUSSION THREAD

**Discussion by lucagessi** [indomus/forum#2604] [Nuove idee, proposte, progetti e guide]
Ciao a tutti.
Mi chiamo Luca, ho un server in Linux a casa con montato home assistant e mi diverto ogni tanto a aggiungere qualcosa.
Mi è venuta l'idea di rilevare le persone sedute sulle sedie per poi implementare qualche automatismo particolare.
In rete ho visto che usano principalmente sensore di pressione, pulsanti meccanici o accelerometri. I primi vanno posti ovviamente sopra la seduta.
Ho fatto alcune prove con un sensore home made che invece andrebbe posto sotto la seduta e risulta ovviamente meno invasivo (non si vede). L'ho testato su diverse sedie e si è dimostrato affidabile. Attualmente è un prototipo ma se c'è interesse mi piacerebbe renderlo più "industrializzato".
Ho preparato un modulo Google a [questo link](https://docs.google.com/forms/d/e/1FAIpQLSfmDNjSB-48s1CWcsjnFQUr3wpiVA3JPfJ7GqLgzUDF_CPZag/viewform), se siete interessati e avete qualche minuto a disposizione mi sarebbe molto utile il vostro parere. Grazie mille

**Comment by snakuzzo** [comment]
Ciao Luca, ho compilato il modulo, ma credo sia giusto anche dare un parere qui.
Ho visto anche io in passato quei progetti di cui parli e sono sempre stato molto scettico, o comunque l'ho sempre ritenuto un accrocchio invasivo e brutto esteticamente.

Pensandoci bene però, potrebbe essere utile sicuramente in casa per casi specifici legati a persone magari anziane e con determinate patologie. La vera utilità però, io la vedrei in attività commerciali tipo bar, ristoranti, etc., e proprio per tale scopo credo possa essere più adatto l'utilizzo del protocollo zigbee.

Quindi, nel complesso sarei contento di leggere qualcosa di più dettagliato sul tuo progetto.

Grazie a te per la condivisione ;)

**Reply by lucagessi** [reply]
Ciao. Grazie per il tuo parere.
A me è venuto in mente per uno scopo banale, cioè accendere le luci della sala sopra il tavolo quando ci sono delle persone sedute. Poi avendo sedie in plastica senza cuscini non ho trovato nessun modo già esistente e ho cercato di utilizzare qualche tecnica nota.
Ho fatto un primo prototipo. Non sto a scendere nei tecnicismi ma a livello di ingombri è simile a qualsiasi trasmettitore zigbee, serve solo l'alloggio per la batteria e il microcontrollore che gestisce sensore e parte radio. Il sensore vero e proprio è molto banale e va attaccato sotto la sedia insieme alla scheda elettronica (con opportuna scatola ovviamente). A grandi linee si guarda come il campo elettrico generato da un PIN del microcontrollore varia. Senza persona ha un comportamento, quando si siede qualcuno varia sensibilmente questa grandezza e si può assumere che la sedia sia occupata.
